src/smooshable.cpp

```diff
@@ -48,6 +50,7 @@ std::pair<Smooshable, Eigen::MatrixXi> Smoosh(const Smooshable& s_a,
   assert(s_a.right_flex() == s_b.left_flex());
   s_out.marginal() = s_a.marginal() * s_b.marginal();
   BinaryMax(s_a.viterbi(), s_b.viterbi(), s_out.viterbi(), viterbi_idx);
+  s_out.scaler() = s_a.scaler() * s_b.scaler();
```

Comment:
Yeah, so for this first iteration of issue #1 I just wanted to do it in terms of probabilities to see if the general approach was ok with you. I can update it so the matrices are logged, and scalers are summed instead of multiplied.